solutions/swb-app/src/environmentTypeConfigRoutes.ts

```diff
@@ -23,9 +23,6 @@ export function setUpEnvTypeConfigRoutes(
   router.post(
     '/environmentTypes/:envTypeId/configurations',
     wrapAsync(async (req: Request, res: Response) => {
-      if (!uuidValidate(req.params.envTypeId)) {
```

Comment:
environment type format is already validated by regex, if envTypeId is written in a wrong format it will not even hit the API